src/components/NewCommentForm.tsx

```diff
@@ -1,37 +1,104 @@
-import React from 'react';
+import React, { FormEvent, useState } from 'react';
+import classNames from 'classnames';
+import { CommentData } from '../types/Comment';
+
+type Props = {
+  onAddComment: (commnet: CommentData) => Promise<void>;
+};
+
+export const NewCommentForm: React.FC<Props> = ({ onAddComment }) => {
+  const [nameInput, setNameInput] = useState('');
+  const [emailInput, setEmailInput] = useState('');
+  const [commentInput, setCommentInput] = useState('');
```

Comment:
It could be one state with object that contain all fields
```
const [newCommentValues, setNewCommentValues] = useState({
  name: '',
  email: '',
  comment: '',
})

```